docs/_docs/pages.md

```diff
@@ -20,7 +20,16 @@ and associated URLs might look like:
 └── contact.html  # => http://example.com/contact.html
 ```
 
-If you have a lot of pages, you can organize them into subfolders. The same subfolders that are used to group your pages in our project's source will exist in the `_site` folder when your site builds.
+If you have a lot of pages, you can organize them into subfolders. The same subfolders that are used to group your pages in your project's source will then exist in the `_site` folder when your site builds. However, when a page has a *different* permalink set in the front matter, the subfolder at `_site` changes accordingly.
+
+```sh
+.
+|-- about.md          # => http://example.com/about.html
+|-- Documentation     # folder containing pages
+    └── Doc1.md       # => http://example.com/Documentation/Doc1.html
```

Comment:
we should not use capitals in a default example, please stick with lowercase.

Reply:
Fixed, thought it may have added clarification. I'm new to github etc so wasn't aware of the standard.